site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -24,6 +24,23 @@
     {% if show_merge_thread_button and core.getUser().getGroup() <= 2 %}
         <a class="btn btn-primary" style="position:absolute;right: 0px;top:3px;display:inline-block;" title="Merge Threads" onclick="$('#merge-threads').css('display', 'block');">Merge Threads</a>
     {% endif %}
+    {% if show_lock_date and core.getUser().getGroup() <= 2 %}
+        <label  for="lock_thread_date" style="position:absolute;right: 0px;display:inline-block;margin: 0 2px"> Lock Thread date
```

Comment:
Capitalizing the first letter of "date" may make it look better.

Reply:
standard do not suggest to use capital letter in names

Reply:
I mean, it's a bit weird for me to see the placeholder is Thread Lock Date but the text next to it is Thread Lock date.